kubernetes/linux/main.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+# please use this instead of adding env vars to bashrc directly
+# usage: setGlobalEnvVar ENABLE_SIDECAR_SCRAPING true
+setGlobalEnvVar() {
```

Comment:
Is this change tested out thoroughly? When David made this change, he wasnt very sure if it was tested out properly. Unless we have a reason to make this change, I dont think this should be included. Please let me know if you think otherwise.

Reply:
Sure. no worries. As long as it is tested out we should be good.